homeassistant/components/justnimbus/config_flow.py

```diff
@@ -12,13 +12,14 @@
 from homeassistant.data_entry_flow import FlowResult
 from homeassistant.helpers import config_validation as cv
 
-from .const import DOMAIN
+from .const import CONF_ZIP_CODE, DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
 STEP_USER_DATA_SCHEMA = vol.Schema(
     {
         vol.Required(CONF_CLIENT_ID): cv.string,
+        vol.Required(CONF_ZIP_CODE): cv.string,
```

Comment:
Existing config entries need to be migrated. Probably start a re-auth flow in `async_setup_entry` and update the config entry data in the re-auth flow in the config flow.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#reauthentication